src/applications/personalization/dashboard-2/components/NotificationCTA.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const NotificationCTA = ({ CTA }) => {
+  const { ariaLabel, href, text, icon } = CTA;
+
+  return (
+    <a
+      aria-label={`${ariaLabel} (opens in new tab)`}
```

Comment:
I realize we should add some tests for this component, but I suggest we should add those in once we hook up the redux flow to fetch actual data.